tensorboard/plugins/histogram/histograms_demo.py

```diff
@@ -18,25 +18,18 @@
 
 from absl import app
 import tensorflow as tf
-
-from tensorboard.plugins.histogram import summary as histogram_summary
+import os
 
 # Directory into which to write tensorboard data.
 LOGDIR = "/tmp/histograms_demo"
 
 
-def run_all(logdir, verbose=False, num_summaries=400):
-    """Generate a bunch of histogram data, and write it to logdir."""
-    del verbose
-
-    tf.compat.v1.set_random_seed(0)
-
-    k = tf.compat.v1.placeholder(tf.float32)
-
+def run(k):
```

Comment:
nit: could we add a docstring that explains what the `k` arg refers to?

Reply:
Done. Thanks for the help!